libs/filamat/src/shaders/MaterialInfo.h

```diff
@@ -33,6 +33,7 @@ using CullingMode = filament::backend::CullingMode;
 
 struct UTILS_PUBLIC MaterialInfo {
     bool isLit;
+    bool hasDoubleSidedCapability;
     bool isDoubleSided;
```

Comment:
Is this still useful?

Reply:
Yes ShaderGenerator needs to know whether or not to emit the `if (doubleSided) {}`.

Reply:
Isn't `hasDoubleSidedCapability` enough for this? Do we need both `isDoubleSided` and `hasDoubleSidedCapability`?

Reply:
Doh thanks, cleaned this up.